django/core/checks/urls.py

```diff
@@ -98,7 +98,11 @@ def check_pattern_startswith_slash(pattern):
     if regex_pattern.startswith('/') or regex_pattern.startswith('^/'):
         warning = Warning(
             "Your URL pattern {} has a regex beginning with a '/'. "
-            "Remove this slash as it is unnecessary.".format(describe_pattern(pattern)),
+            "Remove this slash as it is unnecessary. If this url is "
+            "included, check that the include pattern has a "
```

Comment:
I had some trouble understanding exactly what this meant. Suggestion, "If this pattern is targeted in an include(), ensure the include() pattern has a trailing '/'."

There is also "docs/ref/checks.txt" to update with the new message.